src/common/layout/styles.ts

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+import { getImageUrl } from '@utils/getImageUrl';
+
+export const MenuBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  position: fixed;
+  background-color: #fff1d9;
+  border-right: 7px solid #ffe8c7;
+  height: 100%;
+`;
+
+export const HeaderBox = styled.header`
+  display: flex;
+  justify-content: flex-end;
+  gap: 20px;
+  align-items: center;
+  width: 294px;
+  height: 42px;
+  position: fixed;
+  padding-right: 20px;
+  z-index: 1;
+`;
+
+export const LogoBox = styled.div`
+  margin-top: 26px;
+  margin-left: 39px;
+`;
+
+export const OverRay = styled.div`
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  z-index: 100;
+`;
+
+export const SortContainer = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+`;
+
+export const SortSelectButton = styled.button<{ $isToggled: boolean }>`
+  width: 136px;
+  height: 30px;
+  color: #fff3c0;
+  font-size: 12px;
+  font-weight: 700;
+  text-align: center;
+  background: ${({ $isToggled }) =>
+    $isToggled
+      ? `#d37744 url(${getImageUrl(
+          '정렬-아래-화살표.svg'
+        )}) no-repeat right 10px center`
+      : `#d37744 url(${getImageUrl(
+          '정렬-위-화살표.svg'
+        )}) no-repeat right 10px center`};
```

Comment:
```suggestion
const getSortSelectButtonIcon = (isToggled: boolean) => {
  return isToggled ? '정렬-아래-화살표.svg' : '정렬-위-화살표.svg';
}

// ... 
  background: ${({ $isToggled }) =>
    `#d37744 url(${getSortSelectButtonIcon($isToggled)}) no-repeat right 10px center`;
```
저는 이미지 제외 속성들이 동일하기 때문에 이미지만 가져오는 함수를 분리해서 중복 코드 제거 및 가독성 개선 할 것 같습니다 :)

Reply:
```tsx
const getSortSelectButtonIcon = (isToggled: boolean) => {
  return isToggled ? '정렬-아래-화살표.svg' : '정렬-위-화살표.svg';
};

export const SortSelectButton = styled.button<{ $isToggled: boolean }>`
  width: 136px;
  height: 30px;
  color: #fff3c0;
  font-size: 12px;
  font-weight: 700;
  text-align: center;
  border: 2px solid #c26b3b;

  background: ${({ $isToggled }) =>
    `#d37744 url(${getImageUrl(
      getSortSelectButtonIcon($isToggled)
    )}) no-repeat right 10px center`};
  border-radius: ${({ $isToggled }) => ($isToggled ? '15px 15px 0 0' : '15px')};

  &:focus {
    outline: none;
  }
`;
```
오 이 방법도 상당히 괜찮은 것 같습니다!

Reply:
스타일을 쪼개는 것만 생각했지, 함수를 분리하는건 생각을 못했네요🤣 저도 코드 짤 때 적용해 봐야 겠습니다!